dashboard/app/views/levels/_form.html.haml

```diff
@@ -97,3 +100,17 @@
       }
     }
   });
+
+  $('.expandcollapse').click(function() {
+
+    $('.collapse').each(function(index) {
+        $(this).collapse("toggle");
+    });
+
+    if ($(this).html() == "Expand All") {
+        $(this).html("Collapse All");
+    }
+    else {
```

Comment:
I think there is a js lint error here, which we don't catch in haml files. This file should really be moved into the apps so that it gets linted, transpiled, etc. otherwise, worse things can happen, such as accidentally using es6, which will work for you in Chrome, but then might not work for someone using a different browser.

Can you please extract this JS into a new file? it should go in `apps/src/sites/studio/pages/levels/_form.js`. Here is an example of a different time we created a new js file, so that you can see a couple other hoops to jump through: https://github.com/code-dot-org/code-dot-org/commit/9d4a96faaa8f76d48437348071a2659fd5a8aa48#diff-26cdc4306e5d33ea4df1299321d8852a

Reply:
I noticed one gotcha here, which is that ruby prints some stuff into the js source file like this above: `#{@level.new_record? ? '#new_level' : '.edit_level'}` the way we would normally deal with this is to put the data on the script tag that you use to load this JS, and then pull this off of the script tag from within the JS file. You are probably familiar with this pattern, but here is an example of a commit which adds it: https://github.com/code-dot-org/code-dot-org/commit/054b8bd67f42caead625b72d40ed01ecadc2253e